CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+- Include coex in list of enabled features for docs.rs (#405)
+
+### Fixed
+
+- Small correction to coex warning message (#404)
+- Use a random local port when initializing the wifi stack. (#414)
+
+### Changed
+
+- Update driver blobs (#410)
+
+### Removed
+
```

Comment:
I would probably add two entries for 0.1 and 0.2 and just say initial release supporting chip xyz and wifi/ble etc. Doesn't have to be complete just something there for them